addons/emoji-picker/userscript.js

```diff
@@ -0,0 +1,541 @@
+    },
+    {
+      text: "_binoculars_",
+      image: "/images/emoji/binoculars.png",
+      imager2:
+        "//cdn.scratch.mit.edu/scratchr2/static/__9e4044de46c7852aec750b6571cceb92__/images/easter_eggs/binoculars.png",
+    },
+    {
+      text: "_cupcake_",
+      image: "/images/emoji/cupcake.png",
+      imager2:
+        "//cdn.scratch.mit.edu/scratchr2/static/__9e4044de46c7852aec750b6571cceb92__/images/easter_eggs/cupcake.png",
+    },
+    {
+      text: "_pride_",
+      image: "/images/emoji/pride.png",
+      imager2:
+        "//cdn.scratch.mit.edu/scratchr2/static/__9e4044de46c7852aec750b6571cceb92__/images/easter_eggs/pride.png",
+    },
+    {
+      text: "_blm_",
+      image: "/images/emoji/blm.png",
+      imager2: "//cdn.scratch.mit.edu/scratchr2/static/__9e4044de46c7852aec750b6571cceb92__/images/easter_eggs/blm.png",
+    },
+  ];
+  //Scratch only supports Unicode emojis inside the Basic Multilingual Plane.
+  //This array contains all of those emojis.
+  //"br" is automatically converted to a section break.
+  const unicodeEmojis = [
+    "⌚️",
+    "⏰",
+    "⏱️",
+    "⏲️",
+    "⌨️",
+    "☎️",
+    "⌛️",
+    "⚖️",
+    "⚙️",
+    "✉️",
+    "✂️",
+    "✒️",
+    "☂️",
+    "✏️",
+    "☕️",
+    "♟️",
+    "⚰️",
+    "⚱️",
+    "⛏️",
+    "⚔️",
+    "⚒️",
+    "⛓️",
+    "⚗️",
+    "⛑️",
+    "br",
+    "⭐️",
+    "✨",
+    "⚡️",
+    "☄️",
+    "☀️",
+    "⛅️",
+    "☁️",
+    "⛈️",
+    "⛄️",
+    "❄️",
+    "☔️",
+    "☘️",
+    "br",
+    "☹️",
+    "✌️",
+    "☝️",
+    "✍️",
+    "✋",
+    "☺️",
+    "br",
+    "⚽️",
+    "⚾️",
+    "⛳️",
+    "⛹️",
+    "⛷️",
+    "⛸️",
+    "br",
+    "✈️",
+    "⛵️",
+    "Ⓜ️",
+    "⚓️",
+    "⛽️",
+    "⛲️",
+    "⛺️",
+    "⛪️",
+    "⛰️",
+    "⛱️",
+    "⛴️",
+    "⛩️",
+    "♨️",
+    "br",
+    "❗️",
+    "❕",
+    "❓",
+    "❔",
+    "©️",
+    "®️",
+    "‼️",
+    "⁉️",
+    "™️",
+    "➕",
+    "➖",
+    "➗",
+    "✖️",
+    "⛔",
+    "⭕",
+    "❌",
+    "✔️",
+    "〰️",
+    "〽️",
+    "⚠️",
+    "br",
+    "☑️",
+    "✅",
+    "❎",
+    "▶️",
+    "⏩",
+    "⏪",
+    "⏫",
+    "⏬",
+    "ℹ️",
+    "⏭️",
+    "⏮️",
+    "⏯️",
+    "⏏️",
+    "◀️",
+    "➡️",
+    "⬅️",
+    "⬆️",
+    "⬇️",
+    "↗️",
+    "↘️",
+    "↙️",
+    "↖️",
+    "↪️",
+    "↩️",
+    "⤴️",
+    "⤵️",
+    "✳️",
+    "✴️",
+    "❇️",
+    "㊗️",
+    "㊙️",
+    "br",
+    "⚕️",
+    "☦️",
+    "♾️",
+    "⚛️",
+    "⛎️",
+    "✝️",
+    "☪️",
+    "☮️",
+    "☯️",
+    "☸️",
+    "♈️",
+    "♉️",
+    "♊️",
+    "♋️",
+    "♌️",
+    "♍️",
+    "♎️",
+    "♏️",
+    "♐️",
+    "♑️",
+    "♒️",
+    "♓️",
+    "br",
+    "❤️",
+    "❣️",
+    "♠️",
+    "♣️",
+    "♥️",
+    "♦️",
+    "♀️",
+    "♂️",
+    "♻️",
+    "☢️",
+    "☣️",
+    "⚜️",
+    "➰️",
+    "➿️",
+    "☠️",
+    "br",
+    "⬛️",
+    "⬜️",
+    "⚪️",
+    "⚫️",
+    "▪️",
+    "▫️",
+    "◻️",
+    "◼️",
+    "◽️",
+    "◾️",
+  ];
```

Comment:
You might want to split all this data into a separate js file.

Reply:
Done